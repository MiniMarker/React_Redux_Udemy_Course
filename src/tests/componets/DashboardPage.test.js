import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/DashboardPage';

test("RenderDashboard", () => {
    const wrapper = shallow(<DashboardPage/>);
	expect(wrapper).toMatchSnapshot();
	expect(wrapper.find("Connect(ExpenseListFilters)").length).toBe(1);
	expect(wrapper.find("Connect(ExpenseList)").length).toBe(1)
});