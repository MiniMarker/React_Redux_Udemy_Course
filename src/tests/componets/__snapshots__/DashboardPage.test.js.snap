// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderDashboard 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
